maraboupy/test/test_network.py

```diff
@@ -0,0 +1,58 @@
+# Tests MarabouNetwork features not tested by it's subclasses
+import pytest
+from .. import Marabou
+import os
+import numpy as np
+
+# Global settings
+OPT = Marabou.createOptions(verbosity = 0) # Turn off printing
+TOL = 1e-6                                 # Set tolerance for checking Marabou evaluations
+NETWORK_FOLDER = "../../resources/nnet/"   # Folder for test networks
+
+def test_abs_constraint():
+    """
+    Tests the absolute value constraint.
+    Based on the acas_1_1 test, with abs constraint added to the outputs.
+    """
+    filename =  "acasxu/ACASXU_experimental_v2a_1_1.nnet"
+    testInputs = [
+        [-0.31182839647533234, 0.0, -0.2387324146378273, -0.5, -0.4166666666666667],
+        [-0.16247807039378703, -0.4774648292756546, -0.2387324146378273, -0.3181818181818182, -0.25],
+        [-0.2454504737724233, -0.4774648292756546, 0.0, -0.3181818181818182, 0.0]
+    ]
+    testOutputs = [
+        [abs(0.45556007), 0.44454904, abs(0.49616356), 0.38924966, 0.50136678, abs(testInputs[0][0])],
+        [abs(-0.02158248), -0.01885345, abs(-0.01892334), -0.01892597, -0.01893113, abs(testInputs[1][0])],
+        [abs(0.05990158), 0.05273383, abs(0.10029709), 0.01883183, 0.10521622, abs(testInputs[2][0])]
+    ]
+
+    network = loadNetwork(filename)
+
+    # Replace two output variables with their's absolute value
+    for out in [0, 2]:
+        abs_out = network.getNewVariable()
+        network.outputVars[0][out] = abs_out
```

Comment:
I think switching lines 34 and 35 fixed the problem. Please check and let me know.